keyboards/boardsource/holiday/spooky/config.h

```diff
@@ -1,129 +1,4 @@
-/*
-Copyright 2020 boardsource
-
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation, either version 2 of the License, or
-(at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
-
+// Copyright 2022 @waffle87
```

Comment:
Should be able to delete this file.

Reply:
Was met with numerous errors regarding i2c_master.h without `config_common.h` included. I've opened #17639 as I ran into this with some other keyboards too.